crates/ruff/resources/test/fixtures/flake8_bandit/S201.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/')
+def main():
+    raise
+
+#bad
```

Comment:
For whatever reason, the comment we typically use here is `# Errors` and then `# OK` (instead of `#okay`).